packages/data-model/docs/README.md

```diff
@@ -0,0 +1,119 @@
+
+- `is_valid`: This boolean field indicates whether the submitted data is considered valid based on the schema rules defined in the corresponding dictionary. Tracking the validity status is crucial for ensuring data quality and compliance.
+
+- `last_valid_schema_id` and `original_schema_id`: These fields provide references to the schema versions that were used to validate the submission, supporting effective version control and historical tracking of changes.
+
+- **Timestamps and User Metadata**: Fields such as `created_at`, `created_by`, `updated_at`, and `updated_by` capture important metadata about when and by whom the record was created or modified. This information is essential for maintaining a comprehensive audit trail.
+
+
+
+### - audit_submitted_data
+This table records historical changes to submitted data, providing an audit trail.
+
+Key fields in the audit_submitted_data table include:
+
+- `action`: This field records the type of action performed on the submitted data, represented as an enumerated type (e.g., UPDATE or DELETE)
+
+- `dictionary_category_id`: This field links the audit entry to the corresponding dictionary category associated with the submitted data. It helps contextualize the action within the framework of the defined schema.
+
+- `data_diff`: This field stores a JSON object that captures the differences between the previous and current states of the submitted data. By providing a detailed view of what was changed, this field is essential for tracking modifications and understanding the impact of those changes.
+
+- `entity_name`: This field specifies the name of the entity related to the submitted data, providing additional context for the action logged in the audit entry.
+
+- `last_valid_schema_id`: This field references the schema version that was last validated before the action was taken, ensuring that users can trace back to the applicable schema for the submission.
+
+- `new_data_is_valid`: This boolean field indicates whether the newly submitted data is considered valid according to the schema rules. This status is important for ensuring that only compliant data is retained.
+
+- `old_data_is_valid`: Similar to the previous field, this boolean indicates the validity of the data before the action was taken, allowing for a comparison of data quality over time.
+
+- `organization`: This field identifies the organization responsible for the submission, providing context for the audit entry and aiding in data governance.
+
+- `original_schema_id`: This field references the original schema version used during the initial submission of the data, allowing for a historical perspective on schema changes over time.
+
+- `submission_id`: This field links the audit record to the specific submission, creating a direct connection to the submissin responsible for this action.
```

Comment:
`... connection to the submission responsible for this action.`
Missing 'o' in submission